docs/modernization/WebViewCompatible.md

```diff
@@ -0,0 +1,224 @@
+---
+title: WebViewCompatible control for Windows Forms and WPF
+author: granitestatehacker
+description: The Windows Community Toolkit provides a version of the UWP web view control that can be used in WPF and Windows Forms applications. This control embeds a view into your application that renders web content in one of two ways. For client environments that support the WebViewControl (Windows 10), that implementation is used. For legacy systems, System.Windows.Controls.WebBrowser implements the view.
```

Comment:
Am I right that there is only a compatible view for WPF? Would expect one for both WinForms and WPF